ods_ci/tests/Resources/Page/ODH/ODHDashboard/ResourcesPage.resource

```diff
@@ -90,3 +96,11 @@ Resource Page Should Contain
     END
     Search Resources    search_term=${search_term}
     Verify The Resources Are Filtered    list_of_items=${expected_items}
+
+Verify Resources Count By Checkbox ID
+    [Documentation]    Select resources by Checkbox ID and verify that its count number is correct
+    [Arguments]     ${checkbox_id}
+    Select Checkbox Using Id    ${checkbox_id}
+    ${expected_number}=	   Get Count From Checkbox Label By Id    ${checkbox_id}
```

Comment:
Okay, I have no idea what is this testing exactly, so just a plain ask here - are we sure we are okay to get this dynamically? Don't we want to check that the number of items is changed in the product and thus we should be aware?

Reply:
This KW is used in the "412" Dashbord suite to test the Resources page UI. The logic was duplicated in the 4 tests, so I moved it into this KW, which is now used by those tests. For example, in the test to "Filter Resources By Status Enabled".
Also, since each version the documentation resources may change, and we're only interested to verify that the number of checked items equals to the number of displayed items, then this KW will work for the long term, and fulfill its destiny :)